src/main/resources/assets/computercraft/shaders/monitor.vert

```diff
@@ -0,0 +1,16 @@
+#version 430 core
+
+#define FONT_WIDTH 6.0
+#define FONT_HEIGHT 9.0
+
+layout (location = 0) uniform mat4 u_mv;
+layout (location = 1) uniform mat4 u_p;
+
+layout (location = 0) in vec3 v_pos;
```

Comment:
I don't exactly remember why, but layout location qualifiers are a bit iffy (I think they may not be available on certain versions?). Might be better to use [glBindAttribLocation](https://www.khronos.org/opengl/wiki/GLAPI/glBindAttribLocation) instead.

Reply:
Yep. They're only available on 4.something, so we'll want to remove this.